src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,27 @@
+package controller;
+
+import http.HttpRequest;
+import http.HttpResponse;
+import http.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Controller {
+    protected static final Logger logger = LoggerFactory.getLogger(Controller.class);
+
+    public void service(HttpRequest request, HttpResponse response) {
+        if (request.isGet()) {
+            doGet(request, response);
+        } else if (request.isPost()) {
+            doPost(request, response);
+        }
+    }
+
+    protected void doGet(HttpRequest request, HttpResponse response) {
+        response.setStatus(HttpStatus.NOT_FOUND);
```

Comment:
만약 반환 타입을 `HttpReponse`로 바꾼다면, 아래와 같이 정적 팩토리 메서드로 바로 NotFound 응답을 반환하는 응답 객체를 만들어볼 수 있을 것 같아요. 🙂
```suggestion
        HttpResponse.notFound();
```